advanced_source/cpp_custom_ops.rst

```diff
@@ -174,6 +174,8 @@ To add ``torch.compile`` support for an operator, we must add a FakeTensor kerne
 known as a "meta kernel" or "abstract impl"). FakeTensors are Tensors that have
 metadata (such as shape, dtype, device) but no data: the FakeTensor kernel for an
 operator specifies how to compute the metadata of output tensors given the metadata of input tensors.
+The FakeTensor kernel should return return dummy Tensors of your choice with
+the correct Tensor metadata (shape/strides/``dtype``/device).
```

Comment:
Should this be in double ticks?
```suggestion
the correct Tensor metadata (``shape/strides/<dtype>/device``).
```

Reply:
I just wanted to put dtype in double ticks because it's not an english word